micro-rdk-installer/README.md

```diff
@@ -2,10 +2,16 @@
 
 A CLI that allows a user to flash a build of Micro-RDK, along with their robot's credentials and their wifi information, directly to their esp32 without requiring installation of ESP-IDF, Rust, or Python.
 
+## Download
+
+Find the compiled installer executable corresponding to your system's architecture under the [latest release](https://github.com/viamrobotics/micro-rdk/releases/latest) of micro-RDK.
```

Comment:
This is tricky, I can't seem to get links directly to the files that use "latest" rather than the specific release tag. Do you know of a way around this?

Reply:
nvm, I figured out, sorry